File: webservername-server/src/main/java/com/mycompany/webservername/server/config/DispatcherConfig.java
package com.mycompany.webservername.server.config;

import com.mycompany.webservername.core.config.CoreConfig;
import com.mycompany.webservername.db.config.HibernateConfig;
import org.springframework.context.annotation.ComponentScan;
import org.springframework.context.annotation.Configuration;
import org.springframework.context.annotation.Import;
import org.springframework.web.servlet.config.annotation.EnableWebMvc;
import org.springframework.web.servlet.config.annotation.ResourceHandlerRegistry;
import org.springframework.web.servlet.config.annotation.WebMvcConfigurerAdapter;

/**
 * Created by Artemie on 26.08.2016.
 */
@Configuration
@ComponentScan(basePackages = {
        "com.mycompany.webservername.server.controller",
        "com.mycompany.webservername.server.facade"})
@Import( {AppContextConfig.class, SwaggerConfig.class, CoreConfig.class, HibernateConfig.class})
@EnableWebMvc
public class DispatcherConfig extends WebMvcConfigurerAdapter {

    @Override
    public void addResourceHandlers(ResourceHandlerRegistry registry) {
        registry.addResourceHandler("swagger-ui.html")
                .addResourceLocations("classpath:/META-INF/resources/");

        registry.addResourceHandler("/webjars/**")
                .addResourceLocations("classpath:/META-INF/resources/webjars/");
    }

}

File: webservername-server/src/main/kotlin/com/mycompany/webservername/kotlinbased/server/config/DispatcherConfig.kt
package com.mycompany.webservername.kotlinbased.server.config


import org.springframework.context.annotation.ComponentScan

import org.springframework.context.annotation.Configuration
import org.springframework.context.annotation.Import
import org.springframework.web.servlet.config.annotation.EnableWebMvc
import org.springframework.web.servlet.config.annotation.ResourceHandlerRegistry
import org.springframework.web.servlet.config.annotation.WebMvcConfigurerAdapter

/**
 * Created by Artemie on 26.08.2016.
 */

@Configuration
@ComponentScan(basePackages = arrayOf("com.mycompany.webservername.kotlinbased.server.controller"))
@Import(SwaggerConfig::class)
@EnableWebMvc
open class DispatcherConfig : WebMvcConfigurerAdapter() {

    override fun addResourceHandlers(registry: ResourceHandlerRegistry?) {
        registry!!.addResourceHandler("swagger-ui.html").addResourceLocations("classpath:/META-INF/resources/")
        registry.addResourceHandler("/webjars/**").addResourceLocations("classpath:/META-INF/resources/webjars/")
    }

}
